### PR TITLE
Ikr 4

### PR DESCRIPTION
**Pull request by IlliaKrasheninnikov**
Main updated. The project works (I hope so)

### CODE REVIEW

**Review by Vadim211 — APPROVED** [review]
LGTM

File: Program.cs [hunk 1]
```diff
@@ -58,32 +44,11 @@ static void Main()
 
             var lesson = new Lesson("Ukrainian", "Ukrainian alphabet", edPlatform.Students)
             {
-                Tutor = tutorManager.CreateTutor("Вікторія", "Вакарчук", new DateOnly(2002, 08, 08)),
+                Teacher = tutor_manager.CreateTutor("Вікторія", "Вакарчук", new DateOnly(2002, 08, 08)),
```
Comment (DanyloNaumenko) [inline Program.cs:47]: Could you please create Teacher through the edPlatformManager.SignUp() method

**Review by DanyloNaumenko — CHANGES_REQUESTED** [review]
Good, but please, use the EdPlatformManager Sign Up method to create new Programm users

**Review by DanyloNaumenko — APPROVED** [review]
LGTM

**Pull request merged**